browser/lib/markdown.js

```diff
@@ -2,9 +2,11 @@ import markdownit from 'markdown-it'
 import emoji from 'markdown-it-emoji'
 import math from '@rokt33r/markdown-it-math'
 import _ from 'lodash'
+import ConfigManager from 'browser/main/lib/ConfigManager'
 
 // FIXME We should not depend on global variable.
 const katex = window.katex
+var config = ConfigManager.get()
```

Comment:
Please use `const` instead of `var`.
In a modern browser, please use `let` or `const`.